src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -26,9 +27,11 @@
 	private static void setupPresetTests(Application application) {
 		SelectTestFigureOptionListener selectTestFigureOptionListener = new SelectTestFigureOptionListener(
 				DriverFeature.getDriverManager());
-
 		application.addTest("Figure Joe 1", selectTestFigureOptionListener);
 		application.addTest("Figure Joe 2", selectTestFigureOptionListener);
+		application.addTest("Triangle", event -> FiguresFactory.triangle(DriverFeature.getDriverManager().getCurrentDriver()).execute());
+		application.addTest("Rectangle", event -> FiguresFactory.rectangle(DriverFeature.getDriverManager().getCurrentDriver()).execute());
+		application.addTest("Rhombus", event -> FiguresFactory.rhombus(DriverFeature.getDriverManager().getCurrentDriver()).execute());
```

Comment:
It is good to keep a convention used in the original code (class instance instead of lambda)